Text/AbbyyOCR/AbbyyOCR.cs

```diff
@@ -0,0 +1,257 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using AzureCognitiveSearch.PowerSkills.Common;
+using System.Collections.Generic;
+using Newtonsoft.Json.Linq;
+using Abbyy.CloudSdk.V2.Client.Models;
+using Microsoft.Extensions.DependencyInjection;
+using System.Net.Http;
+using Abbyy.CloudSdk.V2.Client;
+using Abbyy.CloudSdk.V2.Client.Models.RequestParams;
+using Abbyy.CloudSdk.V2.Client.Models.Enums;
+using System.Net;
+using Polly.Extensions.Http;
+using Polly;
+using System.Collections.Concurrent;
+
+namespace AbbyyOCR
+{
+    public static class AbbyyOCR
+    {
+        private const string ApplicationId = @"[Enter ABBYY Application ID]";
+        private const string Password = @"[Enter ABBYY OCR Password]";
+        private const string ServiceUrl = "[Enter ABBYY Service URL such as https://cloud-westus.ocrsdk.com]";
+
+        private static int _retryCount = 3;
+        private static int _delayBetweenRetriesInSeconds = 3;
+        private static string _httpClientName = "OCR_HTTP_CLIENT";
+
+		private static ConcurrentBag<WebApiResponseRecord> bag = new ConcurrentBag<WebApiResponseRecord>();
+
+
+		private static readonly AuthInfo AuthInfo = new AuthInfo
```

Comment:
Please check that you indent with spaces consistently. This has tabs and looks wrongly indented on GitHub. There are other occurrences throughout this file.